command/pr_review.go

```diff
@@ -23,18 +23,24 @@ func init() {
 
 var prReviewCmd = &cobra.Command{
 	Use:   "review [<number> | <url> | <branch>]",
-	Short: "Add a review to a pull request.",
-	Args:  cobra.MaximumNArgs(1),
-	Long: `Add a review to either a specified pull request or the pull request associated with the current branch.
-
-Examples:
-
-	gh pr review                                  # add a review for the current branch's pull request
```

Comment:
Something bothered me about the `branch's` contraction, so I've avoided it by rewording